pkg/reconciler/ingress/status.go

```diff
@@ -0,0 +1,268 @@
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/knative/serving/pkg/network/prober"
+	"github.com/knative/serving/pkg/reconciler/ingress/resources"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"knative.dev/pkg/apis/istio/v1alpha3"
+
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	istiolisters "knative.dev/pkg/client/listers/istio/v1alpha3"
+)
+
+const (
+	probeConcurrency = 15              // defines how many probing calls can be issued simultaneously
```

Comment:
```suggestion
	probeConcurrency = 15              // Defines how many probing calls can be issued simultaneously.
```

Reply:
I don't mind following whatever rule we pick, but we should enforce some comment formatting rules.
Today, it is all over the place in the Knative code base.
I was following the style found in Go codebase: https://golang.org/doc/effective_go.html#formatting

Reply:
I am enforcing full sentences wherever I can (I can live without period in the end :-))